### ISSUE TITLE
Border router dropping packets from tun interface

### ISSUE THREAD

**Issue by tdesmet** [contiki-ng/contiki-ng#811]
My setup is a beaglebone with the native border and slip radio on cc1310 and a californium secure coap server and a client running on the cc1312.

I have a problem where when I restart my secure coap server (Californium) my client is unable to reconnect. 
I can see the client sending CLIENT_HELLO requests and the server receives this and then sends a HELLO_VERIFY_REQUEST but when I sniff the network using sensniff I never see the HELLO_VERIFY_REQUEST.

I then decided to do a dump of the tun interface that is created by the border router and there I see the HELLO_VERIFY_REQUEST packets. The weird thing though is that each HELLO_VERIFY_REQUEST is there two times, once with a hop limit of 64 and once with a hop limit of 63.

I also set the logging of the native border to v3 and there I see the incoming CLIENT_HELLO packets from slip but not the HELLO_VERIFY_REQUEST packet from the tun interface.

```
Jan 08 08:41:55: Packet from SLIP of length 122 - write TUN
Jan 08 08:41:55: Packet input over SLIP: 122
Jan 08 08:41:55: Packet from SLIP of length 45 - write TUN
Jan 08 08:41:55: Packet input over SLIP: 45
```

So to sum up: the server transmits a HELLO_VERIFY_REQUEST packet over the tun interface but the border router seems to drop it...

Attached are some wireshark dumps of the tun interface and of sensniff.
[tun.zip](https://github.com/contiki-ng/contiki-ng/files/2735859/tun.zip)
[sensniff.zip](https://github.com/contiki-ng/contiki-ng/files/2735863/sensniff.zip)

**Comment by tdesmet** [comment]
Hmmm, think I found my problem

When the border router restarts the client device does not send a DAO, so the border router does not have the client device in its routing table and I assume it drops any packets for ip addresses that are not in its routing table?

Is there some guidance on how to solve this? It think this has to do that when I restart the network the instance id and version are the same?

**Comment by tdesmet** [comment]
seems to be related to https://github.com/contiki-ng/contiki-ng/issues/128, closing this
